src/lib/createInlinePluginCreator.ts

```diff
@@ -9,11 +9,11 @@ import {
   Package,
   PluginOption,
   VerifyConditionsContext,
-} from '../typings'
+} from '../typings/index.js'
 
-import getCommitsFiltered from './getCommitsFiltered'
-import { getTagHead } from './git'
-import { resolveReleaseType, updateManifestDeps } from './updateDeps'
+import getCommitsFiltered from './getCommitsFiltered.js'
+import { getTagHead } from './git.js'
+import { resolveReleaseType, updateManifestDeps } from './updateDeps.js'
```

Comment:
why the extensions are needed ?

Reply:
Seems like this is mandatory when using `esm` for binaries. With `import`, node does not tries to resolve the extensions automatically (cause it could be `cjs`, `mjs` or `js`). So now we have to se `.js` everywhere in order to make it work...

Reply:
Weird, in contextualized router I did not have to do this https://github.com/PayFit/express-contextualized-router/blob/master/src/index.ts 🤔

Reply:
Because that's a package. So your package.json will be read to determinate the module resolving to use : 
```
  "main": "dist/index.cjs",
  "module": "dist/index.js",
  "types": "dist/index.d.ts",
```

When using a `bin` the package.json is not used

Reply:
Oh, you're right !